### ISSUE TITLE
Map Ports

### ISSUE THREAD

**Issue by mindbl4ster** [TaskForce47/Liberation#62]
Maps:

- [x] Tanoa

- [ ] Altis

- [x] Chernarus

- [ ] Takistan

- [ ] Reshmaan

- [x] Sahrani

- [ ] Lingor

- [ ] Dingor

- [ ] G.O.S Leskovets

**Comment by jus61** [comment]
TAUNUS ?
https://forums.bistudio.com/topic/185055-wip-terrain-x-cam-taunus/

**Comment by mindbl4ster** [comment]
nope. taunus has an extremly high objectdensity which makes it impossible to play with more than >30 fps with NO units on it.

**Comment by jus61** [comment]
ok thx
